ui/imports/shared/popups/SendModal.qml

```diff
@@ -181,13 +175,17 @@ StatusDialog {
     header: AccountsModalHeader {
         anchors.top: parent.top
         anchors.topMargin: -height - 18
-        model: popup.store.accounts
-        selectedAccount: popup.selectedAccount
-        changeSelectedAccount: function(newAccount, newIndex) {
-            if (newIndex > popup.store.accounts) {
-                return
+        model: SortFilterProxyModel {
+            sourceModel: popup.store.accounts
+            filters: ValueFilter {
+                roleName: "walletType"
+                value: Constants.watchWalletType
+                inverted: true
```

Comment:
The original order was creation time, right? Does it now stay the same for each sub-group (watch-only / normal)?

Reply:
it will still be the same, we are simply removing the watch-only accounts when trying to send tokens cause this actions cannot be performed with a watch only account

Reply:
Oh sorry, misread and thought we were sorting and pushing watch-only accounts to the bottom instead of filtering them. Okay then!